contracts/Rollup.sol

```diff
@@ -1,16 +1,16 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.1;
 
-import '../node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol';
+import '../node_modules/@openzeppelin/contracts/ownership/Ownable.sol';
 import './lib/RollupHelpers.sol';
 import './RollupInterface.sol';
 import './VerifierInterface.sol';
 
 /**
  * @dev Define interface ERC20 contract
  */
-contract ERC20 {
-    function transfer(address recipient, uint256 amount) external returns (bool);
-    function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+abstract contract ERC20 {
```

Comment:
I think it should be `interface ERC20` like in https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/IERC20.sol

Reply:
change it ! xD